### ISSUE TITLE
[FR]: Enhance the Privacy and Security for Download Logs when the user is logged out from the Wallet. 

### ISSUE THREAD

**Issue by ShantanuSharma9873** [KomodoPlatform/komodo-wallet#2446]
**Please describe.**
Currently, the download logs feature allows users who have logged out to download logs without reconnecting their wallet. Moreover, if multiple wallets were previously connected, the logs for all those wallets may be downloaded without logging into any of them.

**Describe your solution**
- Enhance the privacy and security of the download logs feature to ensure that only authenticated and connected wallet users can access their specific logs. 
- Additionally, ensure that logs from multiple previously connected wallets are not accessible without proper authentication.

**Additional context**
Downloaded logs when user is logged out 
[komodo_wallet_log_21.05.2024_20-40-38.txt](https://github.com/KomodoPlatform/komodo-wallet-desktop/files/15392888/komodo_wallet_log_21.05.2024_20-40-38.txt)

**Comment by smk762** [comment]
@ShantanuSharma9873 are you sure this issue is not intended for the web wallet?

**Comment by ShantanuSharma9873** [comment]
Yes @smk762 , this issue is observed on the Web Wallet and still persists.
Please find the attach video attachment and the Downloaded logs when I'm logged out. 
Video - 
https://github.com/user-attachments/assets/9267313a-12f9-4779-bc0d-4d9cab3b2a0c

Downloaded Log - 
[komodo_wallet_log_21.05.2024_20-40-38.txt](https://github.com/user-attachments/files/16695470/komodo_wallet_log_21.05.2024_20-40-38.txt)

**Comment by ShantanuSharma9873** [comment]
The issue still occurs on:
![image](https://github.com/user-attachments/assets/6f41d75e-b0c2-4692-b377-a99e75d8c499)

**Comment by ShantanuSharma9873** [comment]
Please find the Downloaded logs, when the user is logged out.

[komodo_wallet_log_31.10.2024_21-59-15.txt](https://github.com/user-attachments/files/17590508/komodo_wallet_log_31.10.2024_21-59-15.txt)

**Comment by smk762** [comment]
@ShantanuSharma9873 please retest this with 0.9.0

**Comment by ShantanuSharma9873** [comment]
The issue still persists on **version: 0.9.0**
**Screenshot:**
![Image](https://github.com/user-attachments/assets/670ed180-c051-4d21-9ac7-a8440717a017)

**Logs:**
[komodo_wallet_log_25.05.2025_13-46-13.txt.zip](https://github.com/user-attachments/files/20429955/komodo_wallet_log_25.05.2025_13-46-13.txt.zip)
